paddle/fluid/operators/controlflow/CMakeLists.txt

```
@@ -3,21 +3,25 @@ if(WITH_UNITY_BUILD)
   # Load Unity Build rules for operators in paddle/fluid/operators/controlflow.
   include(unity_build_rule.cmake)
 endif()
-register_operators(EXCLUDES conditional_block_op DEPS naive_executor
-                   standalone_executor)
+register_operators(EXCLUDES conditional_block_op static_pylayer_op DEPS
+                   naive_executor standalone_executor)
 
 cc_library(
   conditional_block_op
   SRCS conditional_block_op.cc
   DEPS executor standalone_executor)
+cc_library(
+  static_pylayer_op
+  SRCS static_pylayer_op.cc
+  DEPS executor standalone_executor)
```

```suggestion
  DEPS interpreterCore)
```
应该只需要依赖 interpreterCore 就可以？

---

调用 register_operators 时不需要将 static_pylayer_op 设为 `EXCLUDES` ，则不会发生静态链接时的裁剪。之后将修改 CMakeLists.txt